sources/updater/README.md

```diff
@@ -47,6 +47,8 @@ If the calculated time has not passed yet, Updog returns the update timestamp to
 Assuming all the requirements are met, Updog requests the update images from the TUF repository and writes them to the "inactive" partition.
 
 For more information on what's Updog see [Updog](updog/)
+For more information about update waves see [Waves](waves/)
```

Comment:
Maybe add something like “Updates are rolled out in waves.” Before “For more information about update waves...”

Reply:
The prior paragraph mentions waves. Think that will cover it, or should we add additional context on this line?

Reply:
👍